core/src/types/capability.rs

```diff
@@ -46,72 +46,65 @@ use std::fmt::Debug;
 
-    /// If operator supports read natively, it will be true.
+    /// If operator supports read , it will be true.
     pub read: bool,
-    /// If operator supports seek on returning reader natively, it will
+    /// If operator supports seek on returning reader , it will
     /// be true.
     pub read_can_seek: bool,
-    /// If operator supports next on returning reader natively, it will
+    /// If operator supports next on returning reader , it will
     /// be true.
     pub read_can_next: bool,
-    /// If operator supports read with range natively, it will be true.
+    /// If operator supports read with range , it will be true.
     pub read_with_range: bool,
-    /// If operator supports read with if match natively, it will be true.
+    /// If operator supports read with if match , it will be true.
     pub read_with_if_match: bool,
-    /// If operator supports read with if none match natively, it will be true.
+    /// If operator supports read with if none match , it will be true.
     pub read_with_if_none_match: bool,
-    /// if operator supports read with override cache control natively, it will be true.
+    /// if operator supports read with override cache control , it will be true.
     pub read_with_override_cache_control: bool,
-    /// if operator supports read with override content disposition natively, it will be true.
+    /// if operator supports read with override content disposition , it will be true.
     pub read_with_override_content_disposition: bool,
-    /// if operator supports read with override content type natively, it will be true.
+    /// if operator supports read with override content type , it will be true.
     pub read_with_override_content_type: bool,
 
-    /// If operator supports write natively, it will be true.
+    /// If operator supports write , it will be true.
     pub write: bool,
+    /// If operator supports write by append, it will be true.
+    pub write_can_append: bool,
     /// If operator supports write by sink a stream into, it will be true.
     pub write_can_sink: bool,
     /// If operator supports write with without content length, it will
     /// be true.
     ///
     /// This feature also be called as `Unsized` write or streaming write.
     pub write_without_content_length: bool,
-    /// If operator supports write with content type natively, it will be true.
+    /// If operator supports write with content type , it will be true.
     pub write_with_content_type: bool,
-    /// If operator supports write with content disposition natively, it will be true.
+    /// If operator supports write with content disposition , it will be true.
     pub write_with_content_disposition: bool,
-    /// If operator supports write with cache control natively, it will be true.
+    /// If operator supports write with cache control , it will be true.
     pub write_with_cache_control: bool,
 
-    /// If operator supports append natively, it will be true.
-    pub append: bool,
```

Comment:
It's better to introduce a new capability.

Reply:
Already added.

Reply:
Oh, I see.